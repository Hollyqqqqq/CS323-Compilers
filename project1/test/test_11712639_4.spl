struct node{
    int b = 1;
    a = 10;
};
{
    float f = 0.1;
}