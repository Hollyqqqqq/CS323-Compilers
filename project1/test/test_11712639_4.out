Error type B at Line 3: Missing specifier
Error type B at Line 7: Missing Specifier and FunDec
